modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -340,22 +338,17 @@ public void testAddObjectActionWithConditionExpression() throws Exception {
 			).build(),
 			ServiceContextTestUtil.getServiceContext());
 
-		_objectEntryLocalService.deleteObjectEntry(objectEntry);
-
 		Assert.assertNull(_argumentsList.poll());
 
-		// Add object entry with satisfied condition
+		// Update object entry with satisfied condition
 
-		objectEntry = _objectEntryLocalService.addObjectEntry(
-			TestPropsValues.getUserId(), 0,
-			_objectDefinition.getObjectDefinitionId(),
+		objectEntry = _objectEntryLocalService.updateObjectEntry(
+			TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
 			HashMapBuilder.<String, Serializable>put(
 				"firstName", "João"
 			).build(),
 			ServiceContextTestUtil.getServiceContext());
 
-		objectEntry = _objectEntryLocalService.deleteObjectEntry(objectEntry);
-
```

Comment:
@marcelabc I think we need an updateObjectEntry with `unsatisfied` condition to make sure this feature is working as it should.